src/api/filter.rs

```diff
@@ -0,0 +1,122 @@
+use super::handler::{add_order, get_orders};
+use crate::models::Order;
+use crate::models::OrderBook;
+use warp::Filter;
+
+fn json_body() -> impl Filter<Extract = (Order,), Error = warp::Rejection> + Clone {
+    // When accepting a body, we want a JSON body
+    // (and to reject huge payloads)...
+    warp::body::content_length_limit(1024 * 16).and(warp::body::json())
```

Comment:
nit: make this a constant somwhere?

Reply:
To me, this is not something that we should a) hard code, or b) set in the driver code. To me this kind of setting belongs in our deployment ingress configuration (Nginx settings for example).